Add package whitelist  for auto-cleanup function

thinkhy opened the pull request:
Add an option '--white-list-file' in provider. The packages specified in white list file will not be auto-cleanup between uses.

sorccu commented:
Seems usable for some but we specifically chose to not do something like this at the time STF was built, as we found that on many devices it was possible to uninstall some of the bundled applications but there was no clear way of getting them back once uninstalled. It would have been way too risky in Japan where carrier customization was and is quite heavy.

I’m ok with having this feature, however it should be off by default and should not output a warning message if not used.

thinkhy commented:
@sorccu Thanks for your reviewing. I've just adjusted my code to set the default value of white list file to empty and remove the warning msg. Now the feature is turned off by default and output the msg "Loading the white list file xxx" only if white list file is specified(Non-empty) and existed. Also I added a argument '--white-list-file' in cli/local/index.js to make test and debug easier.
